examples/spring-boot-minimal/build.gradle

```diff
@@ -3,20 +3,27 @@ buildscript {
         gradlePluginPortal()
     }
     dependencies {
-        classpath "org.springframework.boot:spring-boot-gradle-plugin:${managedVersions['org.springframework.boot:spring-boot-gradle-plugin']}"
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:${managedVersions['org.springframework.boot:spring-boot-dependencies']}"
+        classpath "io.spring.gradle:dependency-management-plugin:${managedVersions['io.spring.gradle:dependency-management-plugin']}"
     }
 }
 
 apply plugin: 'org.springframework.boot'
-
-springBoot {
-    mainClassName = 'example.springframework.boot.minimal.Main'
-}
+apply plugin: 'io.spring.dependency-management'
```

Comment:
If I don't add `dependency-management-plugin`, can't compile due to the following error.
Since Spring Boot Gradle Plugin 2.3.0, it has not dependency libraries info.
So I guess we need to add `dependency-management-plugin`.

2.2.7:
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-gradle-plugin/2.2.7.RELEASE

2.3.0:
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-gradle-plugin/2.3.0.RELEASE

```
$ ./gradlew :examples:spring-boot-minimal:build
> Task :examples:spring-boot-minimal:bootJar FAILED

During the build, one or more dependencies that were declared without a version failed to resolve:
    com.google.code.findbugs:jsr305:
    org.jctools:jctools-core:
    org.hibernate.validator:hibernate-validator:
    io.netty:netty-handler-proxy:
    com.github.ben-manes.caffeine:caffeine:
    io.netty:netty-codec-haproxy:
    io.netty:netty-codec-http2:
    org.bouncycastle:bcprov-jdk15on:
    io.netty:netty-tcnative-boringssl-static:
    it.unimi.dsi:fastutil:
    ch.qos.logback:logback-classic:
    io.micrometer:micrometer-core:
    com.google.guava:guava:
    org.reactivestreams:reactive-streams:
    org.slf4j:slf4j-api:
    org.reflections:reflections:
    com.fasterxml.jackson.core:jackson-core:
    javax.inject:javax.inject:
    org.springframework.boot:spring-boot-actuator-autoconfigure:
    com.spotify:completable-futures:
    com.fasterxml.jackson.core:jackson-annotations:
    com.fasterxml.jackson.core:jackson-databind:
    io.netty:netty-resolver-dns:
    org.springframework.boot:spring-boot-starter:
    io.netty:netty-transport:
    org.springframework.boot:spring-boot-autoconfigure:

Did you forget to apply the io.spring.dependency-management plugin to the spring-boot-minimal project?


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':examples:spring-boot-minimal:bootJar'.
> Could not resolve all files for configuration ':examples:spring-boot-minimal:productionRuntimeClasspath'.
   > Could not find com.spotify:completable-futures:.
     Required by:
         project :examples:spring-boot-minimal
         project :examples:spring-boot-minimal > project :core
         project :examples:spring-boot-minimal > project :spring:boot-starter
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback
   > Could not find it.unimi.dsi:fastutil:.
     Required by:
         project :examples:spring-boot-minimal
         project :examples:spring-boot-minimal > project :core
         project :examples:spring-boot-minimal > project :spring:boot-starter
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback
   > Could not find com.google.guava:guava:.
     Required by:
         project :examples:spring-boot-minimal
         project :examples:spring-boot-minimal > project :core
         project :examples:spring-boot-minimal > project :spring:boot-starter
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback
   > Could not find com.google.code.findbugs:jsr305:.
     Required by:
         project :examples:spring-boot-minimal
         project :examples:spring-boot-minimal > project :core
         project :examples:spring-boot-minimal > project :spring:boot-starter
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback
   > Could not find org.jctools:jctools-core:.
     Required by:
         project :examples:spring-boot-minimal
         project :examples:spring-boot-minimal > project :core
         project :examples:spring-boot-minimal > project :spring:boot-starter
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback
   > Could not find org.slf4j:slf4j-api:.
     Required by:
         project :examples:spring-boot-minimal
         project :examples:spring-boot-minimal > project :core
         project :examples:spring-boot-minimal > project :spring:boot-starter
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback
   > Could not find org.reflections:reflections:.
     Required by:
         project :examples:spring-boot-minimal
         project :examples:spring-boot-minimal > project :core
         project :examples:spring-boot-minimal > project :spring:boot-starter
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback
   > Could not find org.hibernate.validator:hibernate-validator:.
     Required by:
         project :examples:spring-boot-minimal
   > Could not find com.fasterxml.jackson.core:jackson-core:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find com.fasterxml.jackson.core:jackson-annotations:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find com.fasterxml.jackson.core:jackson-databind:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.micrometer:micrometer-core:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-transport:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-codec-http2:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-codec-haproxy:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-resolver-dns:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find org.reactivestreams:reactive-streams:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find com.github.ben-manes.caffeine:caffeine:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-transport-native-unix-common:null.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-transport-native-epoll:null.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-tcnative-boringssl-static:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find io.netty:netty-handler-proxy:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find org.bouncycastle:bcprov-jdk15on:.
     Required by:
         project :examples:spring-boot-minimal > project :core
   > Could not find javax.inject:javax.inject:.
     Required by:
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
   > Could not find org.springframework.boot:spring-boot-starter:.
     Required by:
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure
   > Could not find org.springframework.boot:spring-boot-autoconfigure:.
     Required by:
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
   > Could not find org.springframework.boot:spring-boot-actuator-autoconfigure:.
     Required by:
         project :examples:spring-boot-minimal > project :spring:boot-actuator-starter > project :spring:boot-actuator-autoconfigure
   > Could not find ch.qos.logback:logback-classic:.
     Required by:
         project :examples:spring-boot-minimal > project :spring:boot-starter > project :spring:boot-autoconfigure > project :logback

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.4/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 6s
49 actionable tasks: 9 executed, 40 up-to-date
```